manifest.yml

```diff
@@ -1,10 +1,10 @@
 ---
-name: RStudio Server
+name: RStudio Server (Owens Quick)
 category: Interactive Apps
 subcategory: Servers
 role: batch_connect
 description: |
-  This app will launch [RStudio Server] an IDE for [R] on the [Owens cluster].
+  This app will launch [RStudio Server] an IDE for [R] on a single core on a shared and potentially oversubscribed node of the Owens Quick cluster.
```

Comment:
Please change `Owens Quick cluster` to `[Owens cluster]`. I think that it is still worth having the link even if the user no longer has a choice of alternative node types.